x-pack/plugins/ml/public/application/components/annotations/annotation_flyout/index.tsx

```diff
@@ -286,6 +318,23 @@ class AnnotationFlyoutUI extends Component<CommonProps & Props> {
                 value={annotation.annotation}
               />
             </EuiFormRow>
+            <EuiFormRow>
```

Comment:
We should probably add the populated partitioning fields and the detector description in the summary section at the top of the Edit flyout:

![image](https://user-images.githubusercontent.com/7405507/86465121-ebc32e00-bd28-11ea-845d-fbf08d8ed81d.png)